src/map/utils/trustutils.cpp

```diff
@@ -209,56 +211,59 @@ namespace trustutils
                 trust->sJob           = (uint8)sql->GetIntData(6);
                 trust->hasSpellScript = (bool)sql->GetIntData(7);
                 trust->spellList      = (uint16)sql->GetIntData(8);
-                trust->cmbDmgMult     = (uint16)sql->GetIntData(9);
+
+                trust->cmbSkill       = (uint8)sql->GetIntData(9);
                 trust->cmbDelay       = (uint16)sql->GetIntData(10);
-                trust->name_prefix    = (uint8)sql->GetUIntData(11);
-                trust->behaviour      = (uint16)sql->GetUIntData(12);
-                trust->m_MobSkillList = (uint16)sql->GetUIntData(13);
-
-                std::ignore = (uint16)sql->GetUIntData(14); // SpellID
-
-                trust->radius    = sql->GetUIntData(15);
-                trust->EcoSystem = (ECOSYSTEM)sql->GetIntData(16);
-                trust->HPscale   = sql->GetFloatData(17);
-                trust->MPscale   = sql->GetFloatData(18);
-
-                trust->speed     = (uint8)sql->GetIntData(19);
-
-                trust->strRank   = (uint8)sql->GetIntData(20);
-                trust->dexRank   = (uint8)sql->GetIntData(21);
-                trust->vitRank   = (uint8)sql->GetIntData(22);
-                trust->agiRank   = (uint8)sql->GetIntData(23);
-                trust->intRank   = (uint8)sql->GetIntData(24);
-                trust->mndRank   = (uint8)sql->GetIntData(25);
-                trust->chrRank   = (uint8)sql->GetIntData(26);
-                trust->defRank   = (uint8)sql->GetIntData(27);
-                trust->attRank   = (uint8)sql->GetIntData(28);
-                trust->accRank   = (uint8)sql->GetIntData(29);
-                trust->evaRank   = (uint8)sql->GetIntData(30);
+                trust->cmbDmgMult     = (uint16)sql->GetIntData(11);
+
+                trust->name_prefix    = (uint8)sql->GetUIntData(12);
+                trust->behaviour      = (uint16)sql->GetUIntData(13);
+                trust->m_MobSkillList = (uint16)sql->GetUIntData(14);
+
+                std::ignore = (uint16)sql->GetUIntData(15); // SpellID
+
+                trust->radius    = sql->GetUIntData(16);
+                trust->EcoSystem = (ECOSYSTEM)sql->GetIntData(17);
+                trust->HPscale   = sql->GetFloatData(18);
+                trust->MPscale   = sql->GetFloatData(19);
+
+                trust->speed     = (uint8)sql->GetIntData(20);
```

Comment:
ya know we can possibly trim this right out - retail seems to have a static *155* for _all_ of them in client memory, not sure what the packets claim but its unusual to be that value.